### PR TITLE
Random cuisine page

### PR DESCRIPTION
**Pull request by tanyasingh7**
This task is part of User Story #16. 

This is the random cuisine page. On each reload it gives a new cuisine type. Also added a 'different cuisine' button if a user isn't happy with their random cuisine. 
Screen Shot 2021-11-22 at 9 46 28 PM

### CODE REVIEW

File: front-end/src/components/Loading 2.css [hunk 1]
```diff
@@ -0,0 +1,5 @@
+.Loading {
```
Comment (Jen-Lopez) [inline front-end/src/components/Loading 2.css:1]: Seems like there are duplicate loading, request, and error pages. I'd delete these

File: front-end/src/pages/RandomCuisine.jsx [hunk 2]
```diff
@@ -1,23 +1,32 @@
...
+      <div id="random-cuisine-title-top"> {randomCuisine.name}</div>
+      <img id="cuisine-photo" src={require(`../img/cuisines/${randomCuisine.cuisine}/${randomCuisine.thumbnail}`).default} className="food" alt=""></img>
+      <div id="cuisine-text">
+        {randomCuisine.description}
       </div>
-      <div> <h3>Name</h3></div>
-      <div id="cuisine-photo"></div>
-      <div id = "description-container">
-        <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat.</p>
+      <div id="vote-button-div">
```
Comment (Jen-Lopez) [inline front-end/src/pages/RandomCuisine.jsx:24]: can we use the button component instead? Just to make the buttons standardized

File: front-end/src/pages/RandomCuisine.jsx [hunk 3]
```diff
@@ -1,23 +1,32 @@
...
+  const cuisineData = Object.values(data);
+  const generateRand = cuisineData[parseInt(Math.random() * cuisineData.length)];
+  const randomCuisine = generateRand;
+  console.log(randomCuisine);
   return (
     <div className="RandomCuisine">
-      <div id="random-cuisine-title-top"> 
-        <h1>Random Cuisine</h1>
+      
+      <button id="reload" onClick={refreshPage}>Different Cuisine</button>
```
Comment (Jen-Lopez) [inline front-end/src/pages/RandomCuisine.jsx:17]: can we shift this down by the vote button? And perhaps make it the same size? The colors are fine btw

**Review by Golemwardox — APPROVED** [review]
Looks good.

**Review by Jen-Lopez — DISMISSED** [review]
I'd just fix the conflict but looks good!

**Pull request merged**